case/src/main/java/com/ritense/document/service/DocumentDefinitionService.java

```diff
@@ -34,6 +32,8 @@ public interface DocumentDefinitionService {
 
     Page<? extends DocumentDefinition> findAll(Pageable pageable);
 
+    List<JsonSchemaDocumentDefinition> findAllBy(CaseDefinitionId caseDefinitionId);
```

Comment:
Shouldn't this be the following because it's a generic interface?
```suggestion
    List<? extends DocumentDefinition> findAllBy(CaseDefinitionId caseDefinitionId);
```

Reply:
Changed